### PR TITLE
Allow QA role to request all ecosystems

### PR DESCRIPTION
**Pull request by nathanstitt**
Adds:

 * [x] Representers and endpoint for retrieving all ecosystems
 * [x] specs for representers and controller action

Companion to FE PR https://github.com/openstax/tutor-js/pull/747

### CODE REVIEW

File: app/representers/api/v1/ecosystem_representer.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  class EcosystemRepresenter < Roar::Decorator
+    include Roar::JSON
+
+    property :id,
+             type: String,
+             writeable: false,
+             readable: true,
+             schema_info: { required: true }
+
+    property :uuid,
```
Comment (Dantemss) [inline app/representers/api/v1/ecosystem_representer.rb]: I would either nest those inside a :book object or name them something like book_uuid, book_title etc just to be clear.

File: config/secrets.yml.example [hunk 2]
```diff
@@ -11,8 +11,7 @@
 # if you're sharing your code publicly.
 development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] || '8adbae8dca077320646d96808890138f8e9256da4f311eeffde38405ca11b7c0' %>
-  js_url: <%= ENV["JS_URL"] || 'http://localhost:8000/dist/tutor.js' %>
-  css_url: <%= ENV["CSS_URL"] || 'http://localhost:8000/dist/tutor.css' %>
+  app_assets_base_url: <%= ENV["APP_ASSETS_BASE_URL"] || 'http://localhost:8000/dist/' %>
```
Comment (Dantemss) [inline config/secrets.yml.example]: Have to coordinate this change with deployment.

**Comment by Dantemss** [comment]
Oh btw the content analyst stuff is merged, so you can use it. You get a new `is_content_analyst` on the bootstrap data.

**Comment by jpslav** [comment]
Looks great.  Can merge when rebased.

**Comment by nathanstitt** [comment]
rebased

**Comment by joemsak** [comment]
:metal: :smiling_imp:

**Pull request merged**